app/src/main/res/values/dimens.xml

```diff
@@ -75,7 +74,7 @@ http://developer.android.com/guide/topics/appwidgets/index.html#CreatingLayout
 
     <dimen name="app_widget_card_image_size">56dp</dimen>
     <dimen name="app_widget_card_min_width">250dp</dimen>
-    <dimen name="app_widget_card_min_height">64dp</dimen>
+    <dimen name="app_widget_card_min_height">40dp</dimen>
```

Comment:
On my nexus 5, with  the `min_height` set to `64dp` the widget occupied 2 rows. With 40dp it'll be like the other ones and occupied only 1 row